docs/setup.md

```diff
@@ -138,6 +138,9 @@ If you know your way around Rails, here's the very short version. Some additiona
 - Install Gulp (if not already installed)
   - `sudo yarn global add gulp`
 
+- (Optional) Set up a [`post-merge`](https://git-scm.com/docs/githooks#_post_merge) hook to update all dependencies if `package.json` or `Gemfile` changes.
+  - `./git-hooks/install-pull-update-deps`
```

Comment:
I think it has to be `./.git-hooks/install-pull--update-deps` because there's a dot in the folder name.